dspace-server-webapp/src/test/java/org/dspace/app/rest/ItemRestRepositoryIT.java

```diff
@@ -4696,4 +4696,49 @@ public void findAccessStatusForItemTest() throws Exception {
                    .andExpect(jsonPath("$.status", notNullValue()));
     }
 
+    @Test
+    public void findSubmitterTest() throws Exception {
+        context.turnOffAuthorisationSystem();
+
+        //** GIVEN **
+        //1. A community-collection structure with one parent community with sub-community and two collections.
+        parentCommunity = CommunityBuilder.createCommunity(context)
+                .withName("Parent Community")
+                .build();
+        Community child1 = CommunityBuilder.createSubCommunity(context, parentCommunity)
+                .withName("Sub Community")
+                .build();
+        Collection col1 = CollectionBuilder.createCollection(context, child1).withName("Collection 1").build();
+
+        EPerson submitter = EPersonBuilder.createEPerson(context)
+                .withEmail("testone@mail.com")
+                .withPassword(password)
+                .withCanLogin(true)
+                .build();
+
+        context.setCurrentUser(submitter);
+
+        //2. Three public items that are readable by Anonymous with different subjects
+        Item publicItem = ItemBuilder.createItem(context, col1)
+                .withTitle("Public item 1")
+                .withIssueDate("2017-10-17")
+                .withAuthor("Smith, Donald")
+                .withSubject("ExtraEntry")
+                .build();
+
+        context.restoreAuthSystemState();
+
+        String token = getAuthToken(admin.getEmail(), password);
```

Comment:
Can we add a test by a non-admin here too? And confirm result should be 204 No Content (if indeed that is what we want), since the converter will return an empty eperson object when the preauth fails (see my other comment on RestContract: https://github.com/DSpace/RestContract/pull/279#pullrequestreview-2274370852)

Reply:
Done